Subnets.tf

```diff
@@ -0,0 +1,110 @@
+
+resource "aws_subnet" "Geschäftsführung" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.0.0.0/8"
+  availability_zone = "us-west-2a"
+
+  tags = {
+    Name = "Geschäftsführung"
+  }
+}
+
+resource "aws_subnet" "Geschäftsführung-Ersatz" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.16.0.0/8"
+  availability_zone = "us-west-2b"
+
+  tags = {
+    Name = "Geschäftsführung-Ersatz"
+  }
+}
+
+resource "aws_subnet" "Personalabteilung" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.32.0.0/8"
+  availability_zone = "us-west-2a"
+
+  tags = {
+    Name = "Personalabteilung"
+  }
+}
+
+resource "aws_subnet" "Personalabteilung-Ersatz" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.48.0.0/8"
+  availability_zone = "us-west-2b"
+
+  tags = {
+    Name = "Personalabteilung-Ersatz"
+  }
+}
+
+resource "aws_subnet" "Planung" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.64.0.0/8"
+  availability_zone = "us-west-2a"
+
+  tags = {
+    Name = "Planung"
+  }
+}
+
+resource "aws_subnet" "Planung-Ersatz" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.80.0.0/8"
+  availability_zone = "us-west-2b"
+
+  tags = {
+    Name = "Planung-Ersatz"
+  }
+}
+
+resource "aws_subnet" "Personalabteilung-Ersatz" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.48.0.0/8"
+  availability_zone = "us-west-2b"
+
+  tags = {
+    Name = "Personalabteilung-Ersatz"
+  }
+}
+
```

Comment:
```suggestion

```
Maybe Copy/Paste? ;)

Reply:
I have delete the Duplication. I have make so many Subnets because if one Availability_zone go down I have a second Network